dialogue.py

```diff
@@ -7,6 +7,7 @@
 
 import json
 
+
```

Comment:
Do appreciate the reformatting, we could probably do this automatically later.

Reply:
As far as I know, every pull request should consider the code format. My pull requests when I contribute to open source projects always got "changes requested" because of the code format. Since we don't have the standard code format for the team, I prefer to do reformatting before every demo (or some time point) so that everyone feel comfortable looking at the code written by others.